ocaml/networkd/lib/network_device_order.ml

```diff
@@ -262,6 +262,12 @@ module Dev = struct
       )
       devs
 
+  let not_ibft t =
+    try
+      Scanf.sscanf t.name "ibft%d" Fun.id |> ignore ;
```

Comment:
Could use a simpler `String.starts_with ~prefix:"ibft"`; the test above would accept the string below, so is not as accurate as you probably assumed:

```
utop # Scanf.sscanf "ibft123foo" "ibft%d" Fun.id;;
- : int = 123
```

Reply:
Thanks. This is indeed not as expected.